libbroker/broker/internal/peering.hh

```diff
@@ -38,7 +38,7 @@ public:
 
   /// Forces the peering to shut down its connection without performing the BYE
   /// handshake.
-  void force_disconnect();
+  void force_disconnect(const std::string reason);
```

Comment:
This is a sink argument.

```suggestion
  void force_disconnect(std::string reason);
```

Reply:
D'oh, these were silly — thank you. (And, you're welcome to commit to my branches in such cases!)